pytorch/training/docker/1.6.0/py3/cu110/Dockerfile.gpu

```diff
@@ -0,0 +1,192 @@
+# The tag for the base image is: 11.0-cudnn8-devel-ubuntu16.04
+FROM nvidia/cuda:11.0-cudnn8-devel-ubuntu16.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="2"
+
+ARG PYTHON_VERSION=3.6.10
+ARG OPEN_MPI_VERSION=4.0.1
+#ARG CUBLAS_VERSION=10.2.1.243-1_amd64
+ARG CUBLAS_VERSION=11.2.0.252-1_amd64
+
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=0.9.4
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+# Force stdin, stdout and stderr to be totally unbuffered. Good for logging
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/conda/lib"
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+ENV TORCH_CUDA_ARCH_LIST="3.5 3.7 5.2 6.0 6.1 7.0+PTX 8.0"
+ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all"
+ENV HOROVOD_VERSION=0.19.5
+ENV DGLBACKEND=pytorch
+ENV CMAKE_PREFIX_PATH="$(dirname $(which conda))/../"
+ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
+
+ARG PT_TRAINING_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.6.0_aws/20201001-012513/a40b1c195ba679596118514fcbeae1f422591426/gpu/torch-1.6.0-cp36-cp36m-manylinux1_x86_64.whl
+
+RUN apt-get update \
+ && apt-get install -y --allow-change-held-packages --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    cmake \
+    cuda-command-line-tools-11-0 \
+    cuda-cudart-11-0 \
+    libcufft-dev-11-0 \
+    libnccl-dev=2.7.8-1+cuda11.0 \
+    libcublas-dev-11-0=11.2.0.252-1 \
```

Comment:
Do we need both this line as well as the cublas installation from the `.deb` file in the RUN statement after this (line 74)?

Reply:
Addressed